pkg/cephmgr/cephleader.go

```diff
@@ -82,30 +81,30 @@ func (c *cephLeader) HandleRefresh(e *clusterd.RefreshEvent) {
 		// Perform a full refresh of the cluster to ensure the monitors are running with quorum
 		err := c.monLeader.Configure(e.Context, c.factory, c.adminSecret)
 		if err != nil {
-			log.Printf("FAILED TO CONFIGURE CEPH MONS. %v", err)
+			logger.Warningf("FAILED TO CONFIGURE CEPH MONS. %v", err)
```

Comment:
even though this failure (and the next one for osds) don't abort the orchestration i think it better that they be errors instead of warnings